app/src/debug/res/raw/initial_experiments.json

```diff
@@ -81,6 +81,54 @@
           "proposedEnrollment": 7,
           "userFacingDescription": "Experiment to test the home screen configurations",
           "last_modified": 1621443780172
+      },
+      {
+          "slug": "feature-search-term-groups-android",
+          "appId": "org.mozilla.fenix",
+          "appName": "fenix",
+          "channel": "nightly",
+          "branches": [
+              {
+                  "slug": "enabled",
+                  "ratio": 0,
+                  "feature": {
+                      "enabled": true,
+                      "featureId": "search-term-groups"
+                  }
+              },
+              {
+                  "slug": "disabled",
+                  "ratio": 100,
+                  "feature": {
+                      "enabled": false,
+                      "featureId": "search-term-groups"
+                  }
+              }
+          ],
+          "outcomes": [],
+          "arguments": {},
+          "probeSets": [],
+          "startDate": null,
+          "targeting": "true",
+          "featureIds": [
+              "search-term-groups"
+          ],
+          "application": "org.mozilla.firefox_beta",
+          "bucketConfig": {
+              "count": 0,
+              "start": 0,
+              "total": 10000,
```

Comment:
Before this is merged, we want to double check these numbers with Devyani. (`count` can be updated to match `total` if we want all users)

Reply:
to be clearer: this is a ratio of `start` to `count` out of `total`

e.g. 0-100 out of 10000
`start`: 0
`count`: 100
`total`: 10000